apps/client/src/components/Modal.tsx

```diff
@@ -67,17 +69,23 @@ const Modal: React.FC<ModalProps> = ({
     }
   };
 
+  // How to do this properly? @Sven
```

Comment:
Did the solution we discussed work for you?

Reply:
Works perfectly!